draft-ietf-cbor-update-8610-grammar.md

```diff
@@ -178,7 +178,7 @@ sourcecode-name="cddl-new-sesc.abnf"}
 (Notes:
 In ABNF, strings such as `"A"`, `"B"` etc. are case-insensitive, as is
 intended here.
-We could have written `%x62` as `%s"b"`, but didn't, in order to
+The rules above could have written `%x62` as `%s"b"` etc., but didn't, in order to
```

Comment:
```suggestion
The rules above could, instead of `%x62`, also have used `%s"b"` etc., but didn't, in order to
```

While "the rules could have written" is arguably correct, reading it gave me a kneejerk "could have … *been* written" reaction that stopped my reading flow. I'm not sure my suggestion is a real improvement – maybe you find a better rephrasing.